### DISCUSSION TITLE
Particles.js implementation not working on another section! Please help

### DISCUSSION THREAD

**Discussion by husaab** [community/community#102704] [Programming Help]
### Body

Hello! I am trying to add particles.js to another div in my portfolio website. I have already added it to a div in the beginning  but I now want to add it in the experience section. I have tried many things, but it is not working out for me, the div just gets covered in dark navy.

This is the section I want to add it to, my github repository link is here:  https://github.com/husaab/personalwebsite

![image](https://github.com/community/community/assets/126218851/078591e7-3669-4cde-ab73-7623af4952db)

I want the tree to keep its colour, and the boxes, but just a dark navy particles.js background

### Guidelines

- [x] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by DanielWren48** [comment]
Hi @husaab,
I understand what you want this to look like but imho those particles might look out of place and distracting from reading the text from UX stand point.

There is also a style for particles-js
```
#particles-js{
    background-color: #111827;
    position: absolute;
    width: 100%;
    z-index: 0;
    height: 900px;
}
```
that will apply to all ``<div id="particles-js"></div>`` so its kinda a bit more complicated.
So if you definitely want this implemented, let me know and we'll see what can be done.
